q2_feature_table/tests/test_filter.py

```diff
@@ -276,6 +314,11 @@ def test_where(self):
         expected = Table(np.array([]), [], [])
         self.assertEqual(actual, expected)
 
+        # exclude all and no metadata
+        with self.assertRaises(ValueError):
```

Comment:
Can you make this an `assertRaisesRegex` --- I don't think you are catching the `Exception` that you think you are catching (hint: check out the test coverage decrease reported by coveralls). You might want to move this test into a different test group too.